apps/token-e2e/src/integration/flow/governance-flow.cy.js

```diff
@@ -821,13 +816,41 @@ context(
         cy.wait_for_proposal_sync();
         cy.get(proposalDetailsTitle).invoke('text').as('proposalTitle');
         cy.navigate_to('proposals');
-        cy.wait_for_spinner();
       }
 
       function generateProposalTitle() {
         const randomNum = Math.floor(Math.random() * 1000) + 1;
         return randomNum + ': Freeform e2e proposal';
       }
+
+      // This is a workaround function to begin tests with associating tokens without failing
```

Comment:
can we add a ticket to our board in blocked just to keep track of this? I just don't want to lose visibility of it